src-ghc/Pact/PersistPactDb/Regression.hs

```diff
@@ -44,7 +44,7 @@ runRegression p = do
   let mod' = Module "mod1" "mod-admin-keyset" (Meta Nothing Nothing) "code" (H.hash "code") mempty
   _writeRow pactdb Write Modules "mod1" mod' v
   assertEquals' "module write" (Just mod') $ _readRow pactdb Modules "mod1" v
-  assertEquals' "hash of commit 3" (-8810669039792530227) (hashWithSalt 1 <$> commit v)
+  assertEquals' "hash of commit 3" (314509247989860983) (hashWithSalt 1 <$> commit v)
```

Comment:
I'm most concerned about this change. Have I done something wrong somewhere else to cause this?

Reply:
I'm also concerned about this.  Can you check and see if the inputs to the hash function are changing at all as a result of this PR?

Reply:
I'll add a commit to fix this.

Reply:
Great, thanks!